llpc/context/llpcCompiler.cpp

```diff
@@ -2438,6 +2488,14 @@ Result Compiler::buildRayTracingPipelineInternal(RayTracingContext &rtContext,
                                    static_cast<void *>(&helperThreadPayload));
 
     unsigned moduleIndex = 0;
+
+    std::vector<std::thread> workers(cl::AddRtHelpers);
+    for (std::thread &worker : workers) {
+      worker = std::thread([&helperThreadProvider, &helperThreadPayload] {
+        helperThreadBuildRayTracingPipelineElf(helperThreadProvider, &helperThreadPayload);
+      });
+    }
```

Comment:
Maybe move this block to before the `unsigned moduleIndex = 0;` line, to keep that one closer to its usage in `while (!helperThreadPayload.helperThreadJoined && helperThreadProvider->GetNextTask(&moduleIndex)) {`.